naga/src/back/dot/mod.rs

```diff
@@ -453,22 +479,23 @@ fn write_fun(
         for (from, to) in sg.emits {
             writeln!(
                 output,
-                "\t\t{}_s{} -> {}_e{} [ style=dotted ]",
+                "\t\t{}_s{} -> {}_{} [ style=dotted ]",
                 prefix,
                 from,
                 prefix,
-                to.index(),
+                Prefixed(to),
             )?;
         }
     }
 
+    assert!(sg.calls.is_empty());
```

Comment:
Was this accidentally left in?

Reply:
Ohh, hmm, that _is_ weird. @teoxoy, @jimblandy, if `sg.calls` is supposed to be empty, why did we keep code that emits output based on it later? 🤔

Reply:
I missed this while reviewing the PR, it's probably a leftover from debugging an issue.

Reply:
Could you put up a PR for it?